src/deploy/osp_deployer/settings/config.py

```diff
@@ -160,6 +160,39 @@ def __init__(self, settings_file):
         self.tenant_tunnel_vlanid = network_settings[
             'tenant_tunnel_network_vlanid']
         self.tenant_vlan_range = network_settings['tenant_vlan_range']
+        mtu_settings = self.get_settings_section(
+            "MTU Settings")
+        self.mtu_selection = mtu_settings[
+            'mtu_selection']
+        self.mtu_size_global_default = mtu_settings[
+            'mtu_size_global_default']
+        if self.mtu_selection == 'global':
+            self.tenant_tunnel_network_mtu = self.mtu_size_global_default
+            self.tenant_network_mtu = self.mtu_size_global_default
+            self.storage_cluster_network_mtu = self.mtu_size_global_default
+            self.storage_network_mtu = self.mtu_size_global_default
+            self.private_api_network_mtu = self.mtu_size_global_default
+            self.public_api_network_mtu = self.mtu_size_global_default
+            self.floating_ip_network_mtu = self.mtu_size_global_default
+        elif self.mtu_selection == 'per_network':
+            self.mtu_size_global_default = '1500'
+            self.tenant_tunnel_network_mtu = mtu_settings[
+                'tenant_tunnel_network_mtu']
+            self.tenant_network_mtu = mtu_settings[
+                'tenant_network_mtu']
+            self.storage_cluster_network_mtu = mtu_settings[
+                'storage_cluster_network_mtu']
+            self.storage_network_mtu = mtu_settings[
+                'storage_network_mtu']
+            self.private_api_network_mtu = mtu_settings[
+                'private_api_network_mtu']
+            self.public_api_network_mtu = mtu_settings[
+                'public_api_network_mtu']
+            self.floating_ip_network_mtu = mtu_settings[
+                'floating_ip_network_mtu']
+        self.default_bond_mtu = '9000'
```

Comment:
Is there a check somewhere that default_bond_mtu  is compatible with the other MTU's  set?

Reply:
No. The default bond MTU is the maximum allowed MTU value i.e. 9000.